### ISSUE TITLE
Functions missing in structure panel after "while+match" code

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#176]
plugin version: 0.78.2

"missing" should appear in the structure panel.  Perhaps the parser is confused?

### Description

![image](https://user-images.githubusercontent.com/1253341/62012770-59d99b00-b13f-11e9-89fd-0016621d05c1.png)

```
let foo s =
  while true do
    (* "missing" shows in the Structure panel
       if you remove the next 2 lines *)
    match x with
    | _ -> ()
  done

let missing () = Printf.printf "Hi mom!\n"
```

**Comment by giraud** [comment]
fixed in 0.79

**Comment by jfehrle** [comment]
Working for me, thanks!
